documentation/memory/step4/using_native_tools.md

```diff
@@ -1 +1,15 @@
-//TODO
+# Investigation Native memory leaks
+
+While JavaScript is is a garbage collected language, memory is also allocated natively 
+in C/C++ code. These allocations may occur in the Node.js source code, addons or
+in the Node.js dependencies (for example V8).  
+
+[Valgrind](https://valgrind.org/) is a tool that can be used to instrument an
+application such that you get reports on memory usage, including potential memory
+leaks once the application terminates. Valgrind support Linux as well as some variants
+including MacOSX.
+
+We recommend using Valgrind to investigate native memory leaks and more
```

Comment:
If I were not involved in diagnostics, as a reader, I wouldn't be sure who "We" stands for here. I was trying to avoid using it for that reason when doing technical writing not in my name.  
To decide if we ;) should word these recommendations one way or another, a style guide for technical writing would be necessary, so until this happens, my comment is void.  
Just wanted to bring up a styleguide for technical writing, because I've recently seen one being created by a professional and it seems valuable.

Reply:
I kinda agree, but I'd rather not make it harder for folks to write these documents for now, since we're already a small number of folks with limited time.

But I see no problem on suggesting a rewrite here if you want.

(Note that "harder" is subjective. For a technical writer it might be easier to write following a styleguide, but for someone with English as second language and not used to these styleguides, it might be an obstacle)